code/modules/client/preferences2/character_save.dm

```diff
@@ -262,6 +258,10 @@
 		var/datum/species/spath = GLOB.species_list[pick(GLOB.roundstart_races)]
 		pref_species = new spath
 	features = random_features()
+	if(gender == MALE || gender != FEMALE)
+		features["body_model"] = gender
+	else if(gender == PLURAL)
```

Comment:
this if else statement makes 0 sense

Reply:
Removed and replaced it instead with `if(gender)` and now defaults to randomly picking between a male or female body, assuming the species itself has a gender.